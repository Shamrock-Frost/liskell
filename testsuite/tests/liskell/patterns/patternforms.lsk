(defmodule Main _ ())

(define main undefined)

(define (primitivevars a b) undefined)
					;
(define (wildcard _) undefined)

(define (singleConstructor True) undefined)

(define (charLit #\a) undefined)

(define (intLit 3) undefined)

(define (ratLit 3.3) undefined)

(define (stringLit "bar")
    (case (, "kuh" "bar")
      ((, "" "") undefined)))

(define (moreConstructors (Just a) (Just b)) undefined)

(define (asPat (@ asName (Just a))) undefined)

(define (lazyPat (~ (: x xs))) undefined)
