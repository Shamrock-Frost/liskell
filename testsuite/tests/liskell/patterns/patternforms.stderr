[1 of 1] Compiling Main             ( patternforms.lsk, patternforms.o )
Linking patternforms ...
